src/stateless/support_text.vue

```diff
@@ -164,13 +173,13 @@ export default {
         &--left {
             padding-left: 20px;
             padding-right: 10px;
-            transform: translateX(-100%);
+            transform: translateX(-105%);
```

Comment:
Why did we need to change this?

Reply:
Sometimes a few pixels of text container were visible.